Practice Areas/Engineering/code_review_guidelines.md

```diff
@@ -12,12 +12,18 @@ Before sending your code to a VSP team for review, your code should meet all of
 ## Guidelines
 
 * **Review lifecycle**
-    * Before sending code to VSP team, have it reviewed by your team (see **First review** below).
-    * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate to Leah and Patrick.
+    * Before sending code to VSP team have it reviewed by your team (see **First review** below) using a [Draft Pull Request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).  
+      * This prevents [code owners](https://help.github.com/en/articles/about-code-owners) from being notified before pull request is ready for review.
+    * After having your pull request reviewed by your team you can mark your pull request as ready for review.
+      * An engineer from the VSP will aim to review within 24 hours (business days) and each additional round of changes will be reviewed within 24 hours.  If you need it faster, escalate via your DSVA product owner.
+      * Please **do not post in Slack** channels requesting code reviews unless it has been more then 24 hours (business days).
     * After sending your code for review, don't add additional changes! Code needs to be stable for a safe review.
-    * If non-trivial changes to your code are made *after approval by VSP team*, your change is subject to being reverted.
+    * If non-trivial changes to your code are made *after approval by VSP team*, please notify approvers so they can re-review your pull request.
```

Comment:
I like this better, but I'm not sure if we should nix the bit about reverting the change if there was code that was added / changed after the approval.

@gunsch What do you think?

Reply:
Please leave the original wording. The point is to clearly indicate what action VSP team will take if non-trivial changes are submitted after review.

If you want to see that clarified better, I'd prefer:

```suggestion
    * If non-trivial changes to your code are made *after approval by VSP team* and submitted without a re-review, your change is subject to being reverted.
```

Reply:
made suggested change